pkg/injector/pod_patch_test.go

```diff
@@ -281,3 +282,57 @@ func TestAddDaprEnvVarsToContainers(t *testing.T) {
 		})
 	}
 }
+
+func TestGetEnvVarsAnnotation(t *testing.T) {
```

Comment:
@moonchanyong have this test as part of `utils_test.go`.

Reply:
@mukundansundar ok. I added a test to `utils_test.go` and will remove this test.